internal/command/deploy/machines.go

```diff
@@ -191,6 +193,11 @@ func NewMachineDeployment(ctx context.Context, args MachineDeploymentArgs) (Mach
 		maxUnavailable = *appConfig.Deploy.MaxUnavailable
 	}
 
+	bluegreenMaxConcurrent := args.BlueGreenMaxConcurrent
+	if bluegreenMaxConcurrent < 1 {
+		bluegreenMaxConcurrent = 1
+	}
+
```

Comment:
Can we consolidate this down to a single `--max-concurrent` flag? Or do we need a separate bluegreen & immediate flag? For backwards compatibility, we could have `--immediate-max-concurrent` still work but maybe just hide it from the docs?

Reply:
i like that, i'll update